fbfmaproom/fbf-update-data.py

```diff
@@ -397,7 +397,91 @@
     ),
     (
         'mali/chirps-dryspell-jas',
-        'http://iridl.ldeo.columbia.edu/SOURCES/.UCSB/.CHIRPS/.v2p0/.daily-improved/.global/.0p05/.prcp/X/(-12)/(4)/RANGE/Y/(10)/(25)/RANGE/T/(1%20Jun)/61/1/(lt)/0.9/seasonalLLS/T//pointwidth/0/def/(months%20since%201960-01-01)/streamgridunitconvert/T/1.5/shiftGRID/'
+        'http://iridl.ldeo.columbia.edu/SOURCES/.UCSB/.CHIRPS/.v2p0/.daily-improved/.global/.0p05/.prcp/X/(7)/(24)/RANGE/Y/(7)/(24)/RANGE/T/(1%20Jun)/61/1/(lt)/0.9/seasonalLLS/T//pointwidth/0/def/(months%20since%201960-01-01)/streamgridunitconvert/T/1.5/shiftGRID/'
+    ),
```

Comment:
@nitinmagima it looks like instead of adding a new entry for Chad's dryspell dataset you changed the Mali one.

Since you mentioned that Mauritania's dry spell column isn't working, I recommend double-checking all four dry spell URLs.